web/packages/teleport/src/types.ts

```diff
@@ -50,6 +50,7 @@ export enum NavTitle {
   // Access Management
   Users = 'Users',
   Roles = 'User Roles',
+  AccessMonitoring = 'Access Monitoring',
```

Comment:
@ryanclark btw, access monitoring needs to go under the new section i just merged `Identity Governance & Security` line 60